jenkins/README.md

```diff
@@ -46,42 +46,42 @@ jobs on its own.
 ### Cloud Resources cleanup
 
 There is a Jenkins [master job](https://jenkins.nordix.org/view/Airship/job/airship_master_integration_tests_cleanup/)
-that every 6 hours cleans up all the leftover VMs from
-[CityCloud](https://www.citycloud.com/) which fail to be deleted at the end of
-v1alphaX integration test.
+that cleans up all the leftover VMs from
+[CityCloud](https://citycontrolpanel.com/) every 6 hours which has failed to be
+deleted at the end of v1alphaX integration test.
 
 ### "Can one of the admins verify this patch?"
 
 For all the PRs from authors that are not whitelisted, the bot will add a
 comment "*Can one of the admins verify this patch?*". This means that the author
 is not in the whitelist and that someone from the metal3-io organization should
-review the PR
-and run the tests (with */test-integration*) or add the author to whitelist if
-trusted.
+review the PR and run the tests (with */test-integration*) or add the author to
+whitelist if trusted.
 
 ## Jenkins configuration
 
 The jenkins configuration is stored in two places. The Nordix gerrit instance
 contains the jenkins job configuration and the Github metal3-io/project-infra
 repository contains the jobs pipeline.
 
-### job configuration
+### Job configuration
 
-The job configuration is stored [here](https://gerrit.nordix.org/admin/repos/infra/cicd). Please
-announce if some reviews are needed on #cluster-api-baremetal in Kubernetes
+The job configuration is stored [here](https://gerrit.nordix.org/admin/repos/infra/cicd).
+Please announce if some reviews are needed on #cluster-api-baremetal in Kubernetes
 slack or send an email to estjorvas [at] est.tech mailing list.
 
-### job pipeline
+### Job pipeline
 
 The pipeline for the jobs is in the `jobs` folder. The scripts running
 the tests are in the `scripts` folder.
 
 ## Job image
 
-We use pre-baked images to run the tests. The images are based on Ubuntu or
-CentOS.
+We use a volume and pre-baked image (for Ubuntu and Centos, respectively) to run
```

Comment:
Would you mind also to add a sentence to explain what does the volume&pre-baked images do, and why they are needed?

Reply:
Updated